tests/Responses/Expected/promotion_coupon/create_advanced_response.json

```diff
@@ -0,0 +1,18 @@
+{
+    "id": @integer@,
+    "code": "1234",
+    "usageLimit": 10,
+    "used": 0,
+    "expiresAt": "@string@.isDateTime()",
+    "createdAt": "@string@.isDateTime()",
+    "updatedAt": "@string@.isDateTime()",
+    "perCustomerUsageLimit": 1,
+    "_links": {
+        "self": {
+            "href": "\/api\/v1\/promotions\/Holliday\/coupons\/1234"
```

Comment:
If I change that, I'll also have to change it in fixtures which is a duplication of what you've done in https://github.com/Sylius/Sylius/pull/7621 - and would result in conflicts :/